### PR TITLE
fix: failing test

### PR DESCRIPTION
**Pull request by RickWoltheus**
THis pr will increare the timeout a little bit and also add docs for how to run the e2e test locally

Solves PZ-833

### CODE REVIEW

File: docs/development/end-to-end-testing.md [hunk 1]
```diff
@@ -34,3 +34,29 @@ You need to make sure to install the official [cucumber](https://marketplace.vis
 
 Then you will have all the autocomplete features available to you
 
+### Running e2e tests locally
+
+Running e2e tests locally unfortunately requires some extra steps to make it work with our current setup. please don't commit any of these changes, create a stash of it instead so you can easily apply it when needed.
```
[the remark this answers is not in the document]
Comment (RickWoltheus) [reply docs/development/end-to-end-testing.md:39]: You have to change some lines in realms.json and docker-compose.yaml, the changes are visible after this and next sentence. Will make it a bit more clear

File: docs/development/end-to-end-testing.md [hunk 2]
```diff
@@ -34,3 +34,29 @@ You need to make sure to install the official [cucumber](https://marketplace.vis
...
+
+The different docker containers need to be able to communicate with each other. This is not possible when you use localhost. You need to use the host.docker.internal hostname instead.
+
+`docker-compose.yml`
+- replace all the "localhost" with "host.docker.internal"
```
Comment (edgarvonk) [inline docs/development/end-to-end-testing.md:44]: Hmm, dat is strikt genomen niet nodig. Niet alle 'localhosts' hoef je te veranderen. Sterker nog, het ging er maar om eentje volgens mij, waar we vorige week achterkwamen? Ik weet niet meer welke.

File: docs/development/end-to-end-testing.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (edgarvonk) [reply docs/development/end-to-end-testing.md:39]: Those changes in the `realms.json` file can be permanent. I would just add them to this PR. They do not hurt, even if you do not use host.docker.internal. We already have such config settings anyway. See e.g. in the config for Open Zaak in our Docker Compose file:

```- ALLOWED_HOSTS=localhost,host.docker.internal,openzaak.local```

The changes in the hostnames in our Docker Compose file are a different story indeed. We do not want to make those permanent or else e.g. our integration tests on GitHub will fail.

**Review by edgarvonk — CHANGES_REQUESTED** [review]

File: src/e2e/step-definitions/common.ts [hunk 3]
```diff
@@ -12,7 +12,7 @@ Given("{string} navigates to {string} with path {string}", { timeout: 60 * 1000
 });
 
 Then("{string} sees the text: {string}", async function (this: CustomWorld, user, text) {
-    await this.page.waitForSelector(`text=${text}`);
+    await this.page.waitForSelector(`text=${text}`, {'timeout': 10000 });
```
Comment (edgarvonk) [inline src/e2e/step-definitions/common.ts:15]: This 10 seconds right? Seems like a very long time.. I guess so but is this really needed? When I run ZAC locally everything runs pretty quickly.

File: docs/development/end-to-end-testing.md [hunk 4]
```diff
@@ -34,3 +34,48 @@ You need to make sure to install the official [cucumber](https://marketplace.vis
...
+    ```
+2. Add the host.docker.internal Entry:
+   - In the /etc/hosts file, add a new line to link host.docker.internal to the IP address of your host machine. This is typically 127.0.0.1 (localhost).
+   - The entry should look like this:
+    ```csharp
+        # ZAC
+        127.0.0.1 host.docker.internal
+    ```
+   - Save the file and exit the text editor.
+3. Verify the Entry:
```
Comment (edgarvonk) [inline docs/development/end-to-end-testing.md:58]: This verification is not really needed I would say but oh well. I mean: it's just an entry in your hosts file. Nothing special at all.

**Review by edgarvonk — APPROVED** [review]

File: src/e2e/step-definitions/common.ts (hunk 3 above)
Comment (RickWoltheus) [reply src/e2e/step-definitions/common.ts:15]: Agree 10 seconds is pretty long, but this is a change that will fix a flacky test in gh actions. sometimes creating a zaak takes over 5 seconds on test so the test will fail. This change is officially out of scope from this pr but i thought i might as well give test a little bit more room to perform these actions

File: docs/development/end-to-end-testing.md (hunk 4 above)
Comment (edgarvonk) [reply docs/development/end-to-end-testing.md:58]: Ok clear. But just to verify a change in your host file, surely it is not needed to spin up an entire Docker container? Why not just do a `ping host.docker.internal` on your command line?

Comment (edgarvonk) [reply docs/development/end-to-end-testing.md:58]: Never mind: with Docker I guess you are testing if you can access it from inside a running Docker container so I guess that is indeed nice. :-)

File: src/e2e/step-definitions/common.ts (hunk 3 above)
Comment (edgarvonk) [reply src/e2e/step-definitions/common.ts:15]: Ok! Wow, 5 seconds to create a zaak. That is indeed long. Would be good to check out what is going on. What I do know is that on GitHub those GitHub Action VMs are really quite small (as in resources). Nothing like your average MacBook so to speak.. ;-)

**Pull request merged**